.eleventy.js

```diff
@@ -134,6 +182,22 @@ module.exports = function (config) {
     pathPrefix = process.env.BASEURL
   }
 
+  config.addUrlTransform((page) => {
```

Comment:
this feels like a nonstandard way of rewriting the permalinks for these pages. I would expect that these should match what's in the markdown file's frontmatter or be configured using the folder-specific data files, per the 11ty docs. Can you tell me more about this decision?

Reply:
Agreed. It's the first way I figured out how to achieve this. Since I've realized you can achieve the same thing in the 11ty configuration and with 11tydata.js files, which is a much cleaner approach.

Will be updated in a future PR. Closing this in favor of a better PR strategy.